glasswall/libraries/editor/editor.py

```diff
@@ -1150,3 +1150,135 @@ def GW2FileErrorMsg(self, session: int):
         error_message = error_bytes.decode()
 
         return error_message
+
+
+    def GW2GetFileType(self, session: int, file_type_id):
+        """ Retrieve the file type as a string.
+
+        Args:
+            session (int): The session number.
+            file_type_id (int): The file type id.
+
+        Returns:
+            file_type (str): The formal file name for the corresponding file id.
+        """
+        # Validate arg types
+        if not isinstance(session, int):
+            raise TypeError(session)
+
+        # API function declaration
+        self.library.GW2GetFileType.argtypes = [
+            ct.c_size_t,
+            ct.c_size_t,
+            ct.POINTER(ct.c_size_t),
+            ct.POINTER(ct.c_void_p)
+        ]
+
+        # Variable initialisation
+        ct_session = ct.c_size_t(session)
+        ct_file_type = ct.c_size_t(file_type_id)
+        ct_buffer_length = ct.c_size_t(0)
+        ct_buffer = ct.c_void_p()
+
+        # API call
+        status = self.library.GW2GetFileType(
+            ct_session,
+            ct_file_type,
+            ct.byref(ct_buffer_length),            
+            ct.byref(ct_buffer)
+        )
+
+        if status not in successes.success_codes:
```

Comment:
Best practice is to do this (for all occurrences in the script):
```suggestion
        if not status in successes.success_codes:
```